UPDATING.md

```diff
@@ -1777,6 +1777,16 @@ previous one was (project_id, dataset_id, ...) (breaking change)
 - `get_tabledata` returns list of rows instead of API response in dict format. This method is deprecated in
  favor of `list_rows`. (breaking change)
 
+#### `airflow.providers.google.cloud.hooks.cloud_build.CloudBuildHook`
```

Comment:
We should also update `providers/google/CHANGELOG.rst`

Reply:
I'm not quite sure what which section I should update. Is it going to be 5.2.0?

Reply:
I think @potiuk creates the change logs automatically so no need handle it.
This PR forces bumping major release as it has a breaking change